docs/reference/ml/ml-shared.asciidoc

```diff
@@ -507,6 +507,335 @@ tag::data-frame-analytics-stats[]
+end::dfas-eta-growth[]
+
+tag::dfas-feature-bag-fraction[]
+`feature_bag_fraction`::::
+(double)
+The fraction of features that is used when selecting a random bag for each 
+candidate split.
+end::dfas-feature-bag-fraction[]
+
+tag::dfas-gamma[]
+`gamma`::::
+(double)
+Regularization factor to penalize trees with large numbers of nodes.
+end::dfas-gamma[]
+
+tag::dfas-lambda[]
+`lambda`::::
+(double)
+Regularization factor to penalize large leaf weights.
+end::dfas-lambda[]
+
+tag::dfas-max-attempts[]
+`max_attempts_to_add_tree`::::
+(integer)
+If the algorithm fails to determine a non-trivial tree (more than a single 
+leaf), this parameter determines how many of such consecutive failures are 
+tolerated. Once the number of attempts exceeds the threshold, the forest 
+training stops.
+end::dfas-max-attempts[]
+
+tag::dfas-max-optimization-rounds[]
+`max_optimization_rounds_per_hyperparameter`::::
+(integer)
+A multiplier responsible for determining the maximum number of 
+hyperparameter optimization steps in the Bayesian optimization procedure. 
+The maximum number of steps is determined based on the number of undefined hyperparameters 
+times the maximum optimization rounds per hyperparameter.
+end::dfas-max-optimization-rounds[]
+
+tag::dfas-max-trees[]
+`max_trees`::::
+(integer)
+The maximum number of trees in the forest.
+end::dfas-max-trees[]
+
+tag::dfas-num-folds[]
+`num_folds`::::
+(integer)
+The maximum number of folds for the cross-validation procedure.
+end::dfas-num-folds[]
+
+tag::dfas-num-splits[]
+`num_splits_per_feature`::::
+(integer)
+Determines the maximum number of splits for every feature that can occur in a 
+decision tree when the tree is trained.
+end::dfas-num-splits[]
+
+tag::dfas-soft-limit[]
+`soft_tree_depth_limit`::::
+(double)
+Tree depth limit is used for calculating the tree depth penalty. This is a soft 
+limit, it can be exceeded.
+end::dfas-soft-limit[]
+
+tag::dfas-soft-tolerance[]
+`soft_tree_depth_tolerance`::::
+(double)
+Tree depth tolerance is used for calculating the tree depth penalty. This is a 
+soft limit, it can be exceeded.
+end::dfas-soft-tolerance[]
+======
+//End class_hyperparameters
+
+tag::dfas-iteration[]
+`iteration`::::
+(integer)
+The number of iterations on the {dfanalytics-job}.
+end::dfas-iteration[]
+
+tag::dfas-timestamp[]
+`timestamp`::::
+(date)
+The timestamp when the statistics returned in milliseconds since the epoch.
+end::dfas-timestamp[]
+
+//Begin class_timing_stats
+tag::dfas-timing-stats[]
+`timing_stats`::::
+(object)
+An object containing time statistics about the {dfanalytics-job}.
+end::dfas-timing-stats[]
++
+.Properties of `timing_stats`
+[%collapsible%open]
+======
+tag::dfas-timing-stats-elapsed[]
+`elapsed_time`::::
+(integer)
+Runtime of the {dfanalytics-job} in milliseconds.
+end::dfas-timing-stats-elapsed[]
+
+tag::dfas-timing-stats-iteration[]
+`iteration_time`::::
+(integer)
+Runtime of the {dfanalytics-job} since the last iteration in milliseconds.
```

Comment:
It might be simpler to explain this as "Runtime of the latest iteration of the classification analysis in milliseconds"